.ci/scripts/package-docker-snapshot.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+#
+# Build the docker image for the cloudbeat, retag and push it to the given docker registry
+#
+# Arguments:
+# - NEW_TAG, this is the tag for the docker image to be pushed.
+# - NEW_IMAGE, this is the docker image name to be pushed.
+#
+set -euo pipefail
+
+NEW_TAG=${1:?Docker tag is not set}
+NEW_IMAGE=${2:?Docker image is not set}
+
+export PLATFORMS='linux/amd64'
```

Comment:
conflicting with `package.sh` is it ok?

Reply:
They are not conflicting, this one is a wrapper for mage package to build docker & package.sh is for binaries/tar.gz